src/app/services/texts/text.service.ts

```diff
@@ -66,10 +66,15 @@ export class TextService {
 
                 const s = new XMLSerializer();
                 body.content = s.serializeToString(body.content);
-                this.cache.setHtmlCache(textId, body.content.replace(/images\/verk\//g, `${this.apiEndPoint}/${this.appMachineName}/gallery/get/${galleryId}/`));
+                if ( String(body.content).includes('images/verk/http') ) {
+                  body.content = body.content.replace(/images\/verk\//g, '');
+                } else {
+                  body.content = body.content.replace(/images\/verk\//g, `${this.apiEndPoint}/${this.appMachineName}/gallery/get/${galleryId}/`);
+                }
+                this.cache.setHtmlCache(textId, body.content);
```

Comment:
@niklil wouldn't it work if this was outside the if statement?
That way you can also show external images in the text?